python/tvm/relay/op/image/_image.py

```diff
@@ -58,6 +58,38 @@ def compute_resize(attrs, inputs, out_type):
 reg.register_injective_schedule("image.resize")
 
 
+@reg.register_convert_op_layout("image.resize")
+def convert_image_resize(attrs, inputs, tinfos, desired_layouts):
+    """Convert Layout pass registration for image resize op.
+
+    Parameters
+    ----------
+    attrs : tvm.ir.Attrs
+        Attributes of current resize op
+    inputs : list of tvm.relay.Expr
+        The args of the Relay expr to be legalized
+    tinfos : list of types
+        List of input and output types
+    desired_layouts : list of layout strings
+        List of layouts defining our desired
+        layout for the data input.
+
+    Returns
+    -------
+    result : tvm.relay.Expr
+        The transformed expr
+    """
+    # pylint: disable=import-outside-toplevel
+    from tvm import relay
```

Comment:
Is there any reason for lazy import?

Reply:
No, I just based this on other layout conversion functions and they seem to always lazy load inside the function: https://github.com/apache/tvm/blob/2c67d7131bc820481aaaca921f9e76f5709051b3/python/tvm/relay/op/nn/_nn.py#L240 Not sure why actually.

Reply:
I see. I'll let @anijain2305 check if this is necessary.

Reply:
I see. I might have copied it from some other place. @jtuyls Can you please check if you can move the import to the top (or just import the right module like `image` if the relay import fails).

Reply:
@anijain2305 Yes, it works with the import at the top. We can also replace it with `from .image import resize` at the top.